app/ui/src/test/java/com/fsck/k9/activity/compose/RecipientLoaderTest.java

```diff
@@ -47,36 +50,39 @@
             ContactsContract.Data.CONTACT_ID,
             ContactsContract.CommonDataKinds.Nickname.NAME
     };
-    static final String[] PROJECTION_CRYPTO_ADDRESSES = { "address", "uid_address" };
-    static final String[] PROJECTION_CRYPTO_STATUS = { "address", "uid_key_status", "autocrypt_key_status" };
+    static final String[] PROJECTION_CRYPTO_ADDRESSES = {"address", "uid_address"};
```

Comment:
there are a lot of unrelated formatting changes in this file. those make it harder to see what actually changed, please avoid this in the future. If the formatting is annoying in some place, it's best to fix it in a simple to review formatting-only PR

Reply:
Sure, that was just my old habit to use autoformat while coding.